### PR TITLE
fix: accessibility issues homepage

### PR DESCRIPTION
**Pull request by lovelacecoding**
- #30

Altered the following things for the homepage:
* Text in navbar (made it darker for more contrast)
* Text in 'Get your walking buddy' section (made it darker for more contrast) 
*  Text in 'Microsoft Partner' section (made it white instead of dark gray)
*  Text in 'Reimagine your daily commute'  section (made it bold instead of highlighted for more contrast)
*  Added alt text to the commute illustrations
*  Added a lang attribute to the HTML tag for screen readers
*  Added aria-labels to the social media icons for screen readers

### CODE REVIEW

File: pages/home/components/ReImagineYourDailyCommute.tsx [hunk 1]
```diff
@@ -18,31 +18,33 @@ const commuteCards: ICommuteCard[] = [
 		highlight: "shopping",
 		img: shopping.src,
 		bgcolor: "#8DD8D7",
-		textColor: "#67CBC9",
+		description:
+			"illustration of multiple shopping bags in different sizes, shapes and colors.",
```
Comment (ashuvssut) [inline pages/home/components/ReImagineYourDailyCommute.tsx:22]: You don't need to write "illustration of" or "image of"
refer https://youtu.be/-jn9aaNn8_I to write great alt texts!

Your alt texts must have the word `WalkingPal` and rephrase you sentence such that Search Engines can know about how this image related to the website.

In this case I think we should write
"Go for a shopping with your WalkingPal buddy"
> if you can think of a better sentence, you should write that

File: pages/home/components/ReImagineYourDailyCommute.tsx [hunk 2]
```diff
@@ -18,31 +18,33 @@ const commuteCards: ICommuteCard[] = [
...
+		description: "illustration of a big green backpack",
 	},
 	{
 		id: 3,
 		text: "Or just a",
 		highlight: "walk!",
 		img: footstep2.src,
 		bgcolor: "#FF8282",
-		textColor: "#FF5959",
+		description: "illustration of two shoe prints",
```
Comment (ashuvssut) [inline pages/home/components/ReImagineYourDailyCommute.tsx:47]: similarly "Go for morning or evening walks with your WalkingPal buddy"

**Review by ashuvssut — CHANGES_REQUESTED** [review]

**Comment by ashuvssut** [comment]
rest LGTM! Nicely done!!

**Comment by lovelacecoding** [comment]
Changed up the alt text for you! I hope it's fine now 😄

**Comment by ashuvssut** [comment]
Thanks! LGTM

**Pull request merged**